src/quo/foundations/colors.cljs

```diff
@@ -203,6 +203,9 @@
 (def danger-50-opa-30 (alpha danger-50 0.3))
 (def danger-50-opa-40 (alpha danger-50 0.4))
 
+;;60 with transparency
+(def danger-60-opa-10 (alpha danger-60 0.1))
```

Comment:
Please recheck, it looks like background is same in all three themes and we don't this color

Reply:
@Parveshdhull in developer mode it is using `danger-60` although in the normal mode it shows `danger-50`. I'll check with the design team.

Reply:
@Francesca-G can you please confirm if the red background color in dark mode for that component should be `danger-50` or `danger-60` ?

Reply:
yeah there is no danger 60 with opacity - it would be the same as 50 tbh because that's how the color resolving works as specified by designers. It's better to use the resolve-color method as the logic is handled there anyway and then future updates are abstracted to that method/mechanism